browser/src/map/handler/Map.Keyboard.js

```diff
@@ -502,11 +502,8 @@ L.Map.Keyboard = L.Handler.extend({
 		var charCode = ev.charCode;
 		var keyCode = ev.keyCode;
 
-		var DEFAULT =0;
-		//var MAC=1; use this when you encounter a MAC value
-
 		if ((this.modifier == UNOModifier.ALT || this.modifier == UNOModifier.SHIFT + UNOModifier.ALT) &&
-		    keyCode >= this.keyCodes.NUM0[DEFAULT]) {
+		    keyCode >= this.keyCodes.NUM0[0]) {
```

Comment:
This is magic number, best to use labels to avoid confusion what does it mean.
1 was for MAC binding
0 was for non-MAC code

Reply:
I see its declaration repeated few times in many function.
We can instead move it to the top of a file as a "global" variable.

Reply:
I took the chane back. It seems that NUM0 doesn't need to be an array. We can check this later again and possibly turn it into a number from array. It was unnecessary for this PR anyway.